main_withdc.py

```diff
@@ -0,0 +1,104 @@
+"""This runs main and outputs it to discord"""
+import os
+import sys
+import pickle
+import time
+import datetime
+import cv2
+import face_recognition
+import discord
+import functions
+
+CAMINDEX = 0
+SAVE_FILE = "encodings.pickle"
+FONT = cv2.FONT_HERSHEY_SIMPLEX
+FONTSCALE = 1
+TEXTCOLOUR = (255, 0, 0)
+TOLERANCE = 0.4
+if not os.path.exists(SAVE_FILE):
+    data = {"encodings":[],"names":[]}
+    print("No encoding file exists running face detection only")
```

Comment:
switch for logging library after the other PR.

Using a library such as logging is better than print for production because it allows any users/developers to control the output level.